### PR TITLE
Beautify the homepage text

### LINKED ISSUE #405: Beautify the homepage text
Make the text on the homepage, "What’s HACK ON THE HILL? Explore. Build. Empower." text look nicer. Off the top of my head, I think making the "HACK ON THE HILL" text can be made gradient like the number in the banner, but that's just a suggestion!

### PR DESCRIPTION
**Pull request by hannahkendall04**
Fixes #405 

Make HACK ON THE HILL gradient and change the color of catchphrase subtitle.

### CODE REVIEW

File: src/components/HomePage/HothDescription.js [hunk 1]
```diff
@@ -41,8 +43,7 @@ const useStyles = makeStyles(theme => ({
 	catchphrase: {
 		textAlign: 'center',
 		fontSize: '2.5em',
-		fontWeight: theme.typography.fontWeightBold,
-		color: theme.palette.primary.dark,
+		color: '#FF5F96',
```
Comment (katelynsyu) [inline src/components/HomePage/HothDescription.js:46]: let's use `theme.palette.secondary.light` here instead so we don't have a hardcoded hex value

File: src/components/HomePage/HothDescription.js [hunk 2]
```diff
@@ -100,7 +101,7 @@ export default function HothDescription() {
 						Hack on the Hill?
 					</Typography>
 					<Typography variant='h2' className={classes.catchphrase}
-						style={{ fontWeight: 500, paddingBottom: isSmall ? theme.spacing(2) : theme.spacing(8) }}>
+						style={{ fontWeight: 600, paddingBottom: isSmall ? theme.spacing(2) : theme.spacing(8) }}>
```
Comment (katelynsyu) [inline src/components/HomePage/HothDescription.js:104]: Let's delete the `fontWeight` styling here and add `fontWeight: 400` to the `catchphrase` styling under`useStyles`

**Review by katelynsyu — CHANGES_REQUESTED** [review]
Great work! A few nits and then I think we're ready to merge this PR in.

File: src/components/HomePage/HothDescription.js (hunk 1 above)
Comment (hannahkendall04) [reply src/components/HomePage/HothDescription.js:46]: Sounds good!

**Review by katelynsyu — APPROVED** [review]
lgtm! She's beautiful 🤩

**Review by maggieelli — APPROVED** [review]
looks amazing :))

**Pull request merged**